app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java

```diff
@@ -36,13 +36,13 @@
 class ContributorAdapter extends ArrayAdapter<Contributor> {
 
     public ContributorAdapter(Context context) {
-        super(context, R.layout.contributor_item);
+        super(context, R.layout.list_contributor_item);
     }
 
     @Override
     public View getView(int position, View convertView, ViewGroup parent) {
         if (convertView == null) {
-            convertView = View.inflate(getContext(), R.layout.contributor_item, null);
+            convertView = View.inflate(getContext(), R.layout.list_contributor_item, null);
```

Comment:
View.inflate does not have such a method. It is only the `LayoutInflator` I
think.


  [image: --]
Friedger Müffke
[image: http://]about.me/friedger
<http://about.me/friedger>


2015-07-26 17:51 GMT+02:00 Said Tahsin Dane <notifications@github.com>:

> In app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java
> <https://github.com/gdg-x/frisbee/pull/416#discussion_r35494256>:
>
> >      }
> >
> >      @Override
> >      public View getView(int position, View convertView, ViewGroup parent) {
> >          if (convertView == null) {
> > -            convertView = View.inflate(getContext(), R.layout.contributor_item, null);
> > +            convertView = View.inflate(getContext(), R.layout.list_contributor_item, null);
>
> Not this pr s scope but this should use parent, false pattern.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/416/files#r35494256>.
>